### PR TITLE
chore(gitbook): tweak demos and add plugins

### PR DESCRIPTION
**Pull request by Hotell**
plugins additions: 

screen shot 2017-01-10 at 10 39 23 am

in action:

skate-gitbook-plugins

### CODE REVIEW

File: book.json [hunk 1]
```diff
@@ -1,12 +1,28 @@
 {
   "title": "skatejs",
-  "plugins": ["jsfiddle"],
+  "plugins": [
+    "github",
+    "edit-link",
+    "anchors",
+    "jsfiddle"
```
Comment (treshugart) [inline book.json]: I was trying to see how this was presented to the user, but couldn't find a good explanation on the plugin itself. Is there a good example out there?

**Comment by treshugart** [comment]
LGTM for now, but would like to see some jsfilddle examples. I see there's plugins for the various *bin apps. Can always iterate on this if need be.

File: book.json (hunk 1 above)
Comment (Hotell) [inline book.json]: screen shot 2017-01-10 at 10 39 23 am

skate-gitbook-plugins

Comment (Hotell) [inline book.json]: I will add those pics, gif to PR description for further reference

**Pull request merged**